### PR TITLE
Add concurrency view

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/OrchestraView.coffee [hunk 1]
```diff
@@ -94,4 +94,16 @@ OrchestraView = Backbone.View.extend(
...
       path: path.duplicate
     ) if element.get('links')._self_duplicate
+
+  addConcurrency: ->
+    if typeof @currentRankView == 'undefined'
+      @oldRender = @render
+      @currentRankView = 0
+    else
+      @currentRankView = @currentRankView + 1
+    @render = () ->
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/OrchestraView.coffee]: why not add an abstract view than the different channel extend it. In this abstract view you could add this this behavior on the render.

**Comment by itkg-nanne** [comment]
done @itkg-alavieille

File: BackofficeBundle/Resources/public/coffee/widget/AbstractConcurrancyCounter.coffee [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  addConcurrency: ->
+    if typeof @currentRankView == 'undefined'
+      @oldRender = @render
+      @currentRankView = 0
+    else
+      @currentRankView = @currentRankView + 1
+    @render = () ->
+      prototype = Object.getPrototypeOf(@)
+      if @currentRankView == prototype.currentRankView
+        @oldRender.apply this, arguments
```
Comment (SOLenG) [inline BackofficeBundle/Resources/public/coffee/widget/AbstractConcurrancyCounter.coffee]: I'm a little perturbed, if for a any reason `prototype.currentRankView` has a same value with `@currentRankView` which is defined, so `@oldRender` will not be initialised.

Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/coffee/widget/AbstractConcurrancyCounter.coffee]: That's the point, not doing the render ifnot same currentRank after having define oldRender line 8

Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/coffee/widget/AbstractConcurrancyCounter.coffee]: @oldRender is always defined as @currentRankView is a dynamically defined option and is always undefined as starting state.

Comment (SOLenG) [inline BackofficeBundle/Resources/public/coffee/widget/AbstractConcurrancyCounter.coffee]: if it's the case, so it's :+1: for me

**Pull request closed without merging**